Processes/DevRel-Publication-Process.md

```diff
@@ -0,0 +1,169 @@
+# Process: Publish New or Updated Content to the Dev Portal
+
+## Overview
+This document provides instructions on how to publish new or updated content to the EOS Dev Portal (docs.eosnetwork.com).
```

Comment:
You must use a fully qualified domain for external links, such as https://docs.eosnetwork.com instead of docs.eosnetwork.com or the link will think it's internal

Also, Docs are not the "EOS Dev Portal", they are "EOS Documentation"

Reply:
Done

Unable to change the name of the file to "Write, edit, Update Process for Content using GitHub"

Reply:
Try to avoid these long names: "Write, edit, Update Process for Content using GitHub". Choose short, concise names, then use the "Overview" section to expand.